packages/one-app-runner/src/startApp.js

```diff
@@ -127,6 +127,21 @@ const generateModuleMap = (moduleMapUrl) => (moduleMapUrl ? `--module-map-url=${
 
 const generateDebug = (port, useDebug) => (useDebug ? `--inspect=0.0.0.0:${port}` : '');
 
+// Node 12 does not support --dns-result-order or --no-experimental-fetch
+// So we have to remove those flags if the one-app version is less than 5.13.0
+// 5.13.0 is when node 16 was introduced.
+const generateNodeFlags = (appVersion) => {
+  const versionArray = appVersion.split('.');
+  if (versionArray.length > 1) {
+    const majorVersion = Number.parseInt(versionArray[0], 10);
+    const minorVersion = Number.parseInt(versionArray[1], 10);
+    if (majorVersion === 5 && minorVersion < 13) {
```

Comment:
Using `semver` package would be a little cleaner but probably not worth adding another dep.

Reply:
I considered using it, but semver doesn't like dockers use of just major or major + minor.

so `5` is invalid semver. If you try to sanitize using the semver package, it makes it `5.0.0` which isn't correct for how docker operates. 

So it was just easier to do it manually